parser/sql/dialect/oracle/src/main/antlr4/imports/oracle/DMLStatement.g4

```diff
@@ -686,7 +686,8 @@ subquery
 modelExpr
     : (numberLiterals ASTERISK_)? ((measureColumn LBT_ (condition | expr) (COMMA_ (condition | expr))* RBT_) 
     | (aggregationFunction LBT_ (((condition | expr) (COMMA_ (condition | expr))*) | (singleColumnForLoop (COMMA_ singleColumnForLoop)*) | multiColumnForLoop) RBT_) 
-    | analyticFunction) (PLUS_ modelExpr | ASTERISK_ numberLiterals (ASTERISK_ modelExpr)?)?
+    | analyticFunction) ((PLUS_|SLASH_) LP_? modelExpr* RP_? | ASTERISK_ numberLiterals (ASTERISK_ modelExpr)?)?
+    | expr
```

Comment:
Can you add sql parser test case for this rule change?

Reply:
As previously mentioned, the test case has already been added. You can find it in the following link: 
- https://github.com/apache/shardingsphere/blob/master/test/it/parser/src/main/resources/sql/supported/dal/explain.xml#L74-L87

Reply:
Thanks for remind, it's ok now.